src/main/java/me/contaria/seedqueue/SeedQueueEntry.java

```diff
@@ -44,6 +44,7 @@ public class SeedQueueEntry {
     private volatile boolean locked;
     private volatile boolean loaded;
     private volatile boolean discarded;
+    private volatile boolean maxWorldGenerationReached = false;

```

Comment:
Please remove the ` = false`, similar to the other booleans

Reply:
This is uhhh kinda required.... Could i set it to false in the constructor perhaps? cause' we check that immediately when wall loads in order to pause and i think it gets defaulted to `true` or smth? I just got a bug where it started generating all seeds without respecting the `Max Generating Seeds (Wall)` option lol